src/ports/postgres/modules/elastic_net/test/elastic_net_install_check.sql_in

```diff
@@ -840,27 +840,27 @@ SELECT elastic_net_train(
 SELECT * FROM house_en;
 SELECT * FROM house_en_summary;
 
-DROP TABLE if exists house_en, house_en_summary, house_en_cv;
-SELECT elastic_net_train(
-    'lin_housing_wi',
-    'house_en',
-    'y',
-    'x',
-    'gaussian',
-    0.1,
-    0.2,
-    True,
-    NULL,
-    'fista',
-    $$ eta = 2, max_stepsize = 0.5, use_active_set = f,
-       n_folds = 3, validation_result=house_en_cv,
-       n_lambdas = 3, alpha = {0, 0.1, 1},
-       warmup = True, warmup_lambdas = {10, 1, 0.1}
-    $$,
-    NULL,
-    100,
-    1e-6
-);
-SELECT * FROM house_en;
-SELECT * FROM house_en_summary;
-SELECT * FROM house_en_cv;
+-- DROP TABLE if exists house_en, house_en_summary, house_en_cv;
```

Comment:
Please add a comment here (and all other tests commented out) on why this is done. If there is a JIRA that tracks the progress of these fixes then include that here as well.